testing_and_setup/compass/ocean/cosine_bell/RMSE/default/compute_rmse.py

```diff
@@ -1,76 +1,85 @@
+#!/usr/bin/env python
+
```

Comment:
I think it's preferable to have a shebang and run scripts directly (`./compute_rmse.py`) than to call `python compute_rmse.py`.  Since you're calling other scripts in this test case directly, I think it's also good to be consistent.